types/evidence.go

```diff
@@ -47,14 +47,18 @@ var _ Evidence = &DuplicateVoteEvidence{}
 
 // NewDuplicateVoteEvidence creates DuplicateVoteEvidence with right ordering given
 // two conflicting votes. If one of the votes is nil, evidence returned is nil as well
```

Comment:
I think this needs to be updated, it looks like if a vote is missing we now report an error.

Reply:
Thanks - good spot